apps/condo/domains/ticket/access/TicketReportService.js

```diff
@@ -0,0 +1,10 @@
+async function canReadTicketReportWidgetData ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return true
+
+    return { organization: { employees_some: { user: { id: user.id }, isBlocked: false }, deletedAt: null } }
```

Comment:
Not the organization should not be deleted, but employees_some should not be deleted:

`{ organization: { employees_some: { user: { id: user.id }, isBlocked: false, deletedAt: null } }`

Otherwise the deleted user will be able to read ticket report widget data

Reply:
See https://github.com/open-condo-software/condo/pull/233

Reply:
First time misunderstood, corrected